xarray/backends/pydap_.py

```diff
@@ -15,16 +15,13 @@ class PydapArrayWrapper(BackendArray):
     def __init__(self, array):
         self.array = array
 
+    @property
+    def shape(self):
+        return self.array.shape
+
     @property
     def dtype(self):
-        t = self.array.type
```

Comment:
`pydap.model.GridType` seems to have no `type` attribute.
Something was changed in pydap?